lms/envs/common.py

```diff
@@ -3009,6 +3009,8 @@ def _make_locale_paths(settings):
 # visible. We default this to the legacy permission 'see_exists'.
 COURSE_ABOUT_VISIBILITY_PERMISSION = 'see_exists'
 
+# Set default course visibility in catalog
```

Comment:
This comment seems to state the obvious.  Either remove this comment or replace it with information that explains why the default is set to `both`.